pairtaghighlighter/src/pair_tag_highlighter.c

```diff
@@ -319,6 +321,9 @@ static void findMatchingTag(ScintillaObject *sci, gint openingBracket, gint clos
     gboolean isTagOpening = is_tag_opening(sci, openingBracket);
     gchar *tagName = get_tag_name(sci, openingBracket, closingBracket, isTagOpening);
 
+    if (!tagName)
```

Comment:
Why I am saying it is redundant - in current implementation of get_tag_name NULL will be returned on empty tag. And check on line 369 preserves from it (marked with comment "Don't run search on empty brackets").

But I don't mind having extra safety check, especially if something changes in future.